CMSgov/mint-app
# [MINT-3006] Refactor Backend to Support Model Plan Phases

# MINT-3006

## Description

In order to support frontend and email template needs we have anticipatedStatuses to include a phase enumeration value for easy state handling. Also, in order to more easily infer attribute intent we have updated the name from anticipatedStatuses to suggestedPhase. A null phase result indicates that there is no currently suggested phase.

## How to test this change

1. Open Postman and run any of the endpoints to validate a phase and corresponding suggested statuses
2. Check unit tests

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
